update-common/src/errors.rs

```diff
@@ -140,6 +140,8 @@ pub enum RepositoryError {
         "duplicate hash entries found in artifacts.json for kind `{}`, hash `{}`", .0.kind, .0.hash
     )]
     DuplicateHashEntry(ArtifactHashId),
+    #[error("error creating reader stream")]
+    CreateReaderStream(#[source] anyhow::Error),
```

Comment:
Not sure the opinion on adding new errors here

Reply:
Adding new errors (when needed) is good :+1: